metaparser/metaparser.go

```diff
@@ -0,0 +1,53 @@
+package metaparser
+
+import (
+	"time"
+
+	"github.com/bitrise-io/go-utils/v2/fileutil"
+	"github.com/bitrise-io/go-utils/v2/log"
+	"github.com/bitrise-io/go-xcode/exportoptions"
+)
+
+// ArtifactMetadata ...
+type ArtifactMetadata struct {
+	AppInfo          Info          `json:"app_info"`
+	FileSizeBytes    int64         `json:"file_size_bytes"`
+	ProvisioningInfo ProvisionInfo `json:"provisioning_info,omitempty"`
+	Scheme           string        `json:"scheme,omitempty"`
+}
+
+// Info ...
+type Info struct {
+	AppTitle          string   `json:"app_title"`
+	BundleID          string   `json:"bundle_id"`
+	Version           string   `json:"version"`
+	BuildNumber       string   `json:"build_number"`
+	MinOSVersion      string   `json:"min_OS_version"`
+	DeviceFamilyList  []uint64 `json:"device_family_list"`
+	RawPackageContent string   `json:"-"`
```

Comment:
Is this filed used?

Reply:
In case of Android Artifacts, this field is used during the validation process. I'm not sure, if we need this for xcode lib too.

Reply:
I suggest removing this field, because it is unused and not playing a role in the JSON models we upload along the artifacts.